component/modal/modal.css

```diff
@@ -0,0 +1,37 @@
+.modal-container{
+  
+    max-width: 600px;
+    width: 100%;
+    padding:1rem;
+    overflow: auto;
+    padding: 0px 30px;
+    box-sizing: border-box;
+    background-color:var(--dark-blue);
+    color:white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+}
+.modal-content-ctn{
+    text-align: center;
+}
+.modal-content-ctn h6{
+    text-align: center;
+}
```

Comment:
Here if the user uses any other headline tags like h1, it would not center it. Can you make it generic?

Reply:
made that generic !!

Reply:
Did u add a commit ?

Reply:
I have just removed this CSS itself as it is not needed .

Reply:
Ok 😊